doc/help/EXPORT.md

```diff
@@ -0,0 +1,30 @@
+kdb-export(1) -- Export keys from the key database
+==================================================
+## DESCRIPTION
+
+This command allows a user to export keys from the key database.  
+Keys are exported to `stdout` in whichever format is specified.  
+The default format is `dump` but can be changed by editing the value of the `sw/kdb/current/format` key.  
+This command can also be used to view full key(s) including their values.  
+
+## USAGE
+
+`kdb export <source> [<format>]`  
+
+Where `source` is the path of the key(s) you want to export.  
+Additionally, the user can specify a format to use by passing it as the option argument `format`.  
```

Comment:
also refer to `kdb list` to list all plugins (+ explain which ones are suitable to be as format)